core/embed/trezorhal/stm32f4/backlight_pwm.c

```diff
@@ -16,6 +18,35 @@ static int pwm_period = 0;
 
 int backlight_pwm_set(int val) {
   if (BACKLIGHT != val && val >= 0 && val <= 255) {
+    // TPS61043: min 1% duty cycle
```

Comment:
The condition here should be something like `val < (255 / 100) + 1`.

In my opinion, we might not need this check. According to the datasheet:

> The CTRL pin accepts a PWM duty cycle from D = 1% to 100%. Duty cycles below 1% are also possible, with the restriction that the device is forced into shutdown if the off time of the applied PWM signal exceeds 10 ms.

Reply:
removed: https://github.com/trezor/trezor-firmware/commit/78a5204dd68a49dadf6dd3b73d2f65698d7ba3c8 and  https://github.com/trezor/trezor-firmware/commit/3444eb464405929e4a2b04de55d01db90656bd90